### ISSUE TITLE
`VideoController.mixWithOthers` has no effect on iOS

### ISSUE THREAD

**Issue by AlexV525** [wang-bin/fvp#172]
**Describe the bug**

When using `fvp` on iOS, setting `mixWithOthers` in the `VideoPlayerController` will still cause the background playback to stop when calling the `VideoPlayerController.initialize` or `VideoPlayerController.play`.

Also,  playing background audio when the video is playing will cause the video to run into a very slow framerate.

**Logs**
[mdk.log](https://github.com/user-attachments/files/17520087/mdk.log)

**Comment by wang-bin** [comment]
try master branch

**Comment by AlexV525** [comment]
```console
Error (Xcode): ../../../../.pub-cache/git/fvp-8281ae0ab6057e6665d181fb7ed1ebd99da06f71/lib/src/video_player_mdk.dart:344:26: Error: The method 'setMixWithOthers' isn't defined for the class 'FvpPlatform'.
```

**Comment by AlexV525** [comment]
https://github.com/wang-bin/fvp/actions/runs/11557830755/job/32168767278

**Comment by AlexV525** [comment]
How about Android and other platforms?

**Comment by wang-bin** [comment]
> ```console
> Error (Xcode): ../../../../.pub-cache/git/fvp-8281ae0ab6057e6665d181fb7ed1ebd99da06f71/lib/src/video_player_mdk.dart:344:26: Error: The method 'setMixWithOthers' isn't defined for the class 'FvpPlatform'.
> ```

i forgot to commit some files. try master branch again

**Comment by AlexV525** [comment]
> i forgot to commit some files. try master branch again

The code is working.

**Comment by wang-bin** [comment]
> How about Android and other platforms (I know that macOS has nothing to do with the playback mode support)?

android should be similar. i guess desktop platforms have no such feature

**Comment by AlexV525** [comment]
So I guess the remain thing is to implement this with the Android platform.

**Comment by wang-bin** [comment]
> So I guess the remain thing is to implement this with the Android platform.

yes, but i don't have enough time recently

**Comment by AlexV525** [comment]
Is the repo open for pull requests? If so I can try to add the functionality.

**Comment by wang-bin** [comment]
yes

**Comment by AlexV525** [comment]
I took some investigation with Android. `video_player` deals with the audio session within the `ExoPlayer`, but we don't seem to have such an interface to control that behavior. The video's audio track is always mixed with other audio on Android, so we should implement the contrary. I don't think there is anything I can do further.

**Comment by wang-bin** [comment]
same as #72, audio_session package support this feature

**Comment by AlexV525** [comment]
> same as #72, audio_session package support this feature

Absolutely! But it would be better to perform the same behavior as the `video_package` since the library declares itself as a plugin set.

**Comment by wang-bin** [comment]
yes, so that issue is still open. we can use audio_session's implementation
